### ISSUE TITLE
rviz_rendering_tests.mesh_loader_test_target failing on nightly_linux-armhf_release

### ISSUE THREAD

**Issue by emersonknapp** [ros2/rviz#506]
This test has been failing consistently for a while in the nightly release armhf job

Test: `rviz_rendering_tests.mesh_loader_test_target`

https://ci.ros2.org/view/nightly/job/nightly_linux-armhf_release/368/#showFailuresLink

Seems to have been failing since at least build#166 in July 2019 - hard to parse the results earlier than that https://ci.ros2.org/view/nightly/job/nightly_linux-armhf_release/166/#showFailuresLink

**Comment by clalancette** [comment]
Since we no longer build on armhf, I'm going to close this out.  If we ever bring it back, or otherwise see it, we can open a new issue.
